src/components/va-back-to-top/va-back-to-top.css

```diff
@@ -0,0 +1,101 @@
+@import '../../../node_modules/animate.css/source/sliding_entrances/slideInUp.css';
+@import '../../../node_modules/animate.css/source/sliding_exits/slideOutDown.css';
```

Comment:
I don't like importing like this, but I couldn't get `postcss-import` to work. Regardless, these get inlined anyway.

At some point I'd like to use postcss to help with our CSS.

Reply:
I like them, but are these animations necessary? Are they accessible? cc @noahgelman

Reply:
Well, they aren't _necessary_. @GnatalieH said she was ok with them as part of this redesign though, since the same results are in the current version (even though this implementation is different).

Reply:
@cvalarida , I can chime in here as Noah is out sick at the moment-- animations could put folks with vestibular disorders at risk, so we'd recommend either (a) not including them in the redesign or (b) using the reduced motion css media query so it only activates when folks have an explicit preference for motion set

Reply:
This ticket on [accessibility for smooth scrolling](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/312) might be helpful

Reply:
I'd recommend the first as (a) it's the easiest and (b) it's less distracting to users 🙏